### PR TITLE
Capstone final: React App with API Integration

### PR DESCRIPTION
**Pull request by MarcoDDM**
This pull request contains the following updates:

1. **API Integration**
   - Researched and selected the appropriate API for the project.
   - Implemented the necessary code to fetch data from the API.
   - Integrated the API response to populate a list of items in the application.

2. **Application Structure**
   - Created a well-organized file structure for the React app.
   - Separated components into appropriate directories for better maintainability.

3. **Routing and Navigation**
   - Set up routes and navigation within the application.

4. **Test Coverage**
   - Created comprehensive tests for all major functionalities of the application.
   - Ensured that the application functions as intended and edge cases are handled properly.

5. **Styling and Design**
   - Styled the components to match the provided design guidelines.
   - Ensured consistency in visual elements and user experience.
   
6. **Deployment and Final Testing**
   - Deployed the project to a suitable hosting platform for easy access and review.
   - Performed thorough testing to ensure all features and functionalities work seamlessly in the production environment.
   
7. **Project Documentation**
   - Recorded a video showcasing the application's features and functionality.
   - Created a detailed README file that includes instructions on how to set up and run the project locally.
   - Crafted a compelling and informative Pull Request description to provide an overview of the implemented solution.

Thank you for your review and don't hesitate to tag me if you need more information or there are any changes involved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+<div align="center">
+
+  <img src="stock_stats.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+**Metric Web App** is a project to show API data on a SPA.
+![Video](https://cdn.loom.com/sessions/thumbnails/dec4f57c759b400e8c4e7f14cb1df71e-with-play.gif)
```
Comment (IngDavidTM) [inline README.md:45]: - [ ] Your video for the project is truly exceptional :+1:, showcasing your remarkable skills. Nevertheless, it is essential to include a highlight in the video that focuses on a specific portion of the code which you personally find intriguing or take pride in creating. This requirement serves the purpose of highlighting your expertise and emphasizing the significance of a particular code segment within the overall project.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @MarcoDDM  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MarcoDDM) [reply README.md:45]: Thank you for your suggestion, I will apply that now...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
```
Comment (ishimwezachee) [inline README.md:45]: - [ ]  You did a great job updating the presentation videos, but it seems like you haven't addressed the suggestions made by the previous reviewer. To clarify their request, when they mentioned highlighting the portion of the code, it means that you should show the code on your videos as well. This entails sharing your screen and displaying the parts of the code that you are proud of working on. I hope this feedback doesn't discourage you. Perhaps it was a minor misunderstanding, but you can definitely do it. Just include that part in your videos, and you'll be ready to go.
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @MarcoDDM , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image

 ## Highlights
improved the video's presentation ✅
The PR title and description are professional and descriptive ✅
Linters are working properly ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @MarcoDDM, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct git flow ✔️ 
- Descriptive PR ✔️ 
- Correct use of Hooks ✔️ 
- Professional documentation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
